runtime/wasm/src/host.rs

```diff
@@ -275,7 +275,7 @@ impl RuntimeHost {
                     };
 
                     result_sender
-                        .send(result)
+                        .send((result, future::ok(Instant::now())))
```

Comment:
Does this need to be a future?

Reply:
Having this be `IntoFuture` lets us keep the rest of the code untouched, so the options were either `Result<Instant, Error>` (being always `Ok`) or `futures::Finished<Instant, Error>` which I went with. This log is a temporary thing anyhow, once we improve things and get less contention warnings, this will be overkill.

Reply:
Makes sense. The reason I'm asking about the future is that I wonder if the `wait` later could add undesired extra time to the measurement.

Reply:
Yhea `.wait()` sounds like it's waiting but that will actually be instant. I improved the comment.